### PR TITLE
Visualisation corrections

### PR DESCRIPTION
**Pull request by zefciu**
* Handled empty positions
* Serializing the admin url for DC

### CODE REVIEW

File: src/ralph_assets/models_assets.py [hunk 1]
```diff
@@ -975,16 +975,17 @@ def get_related_assets(self):
...
+            """Returns the integer part of slot number"""
+            m = re.match('(\d+)', slot_no)
+            return (m and int(m.group(0))) or 0
+
+        max_slot_no = max([
+            get_number(asset.device_info.slot_no)
+            for asset in assets
+        ])
+        first_asset_slot_no = assets[0].device_info.slot_no
+        ab = first_asset_slot_no and first_asset_slot_no[-1] in {'A', 'B'}
```
Comment (xliiv) [inline src/ralph_assets/models_assets.py]: `is_ab`?

**Comment by xliiv** [comment]
:ok:

**Comment by ar4s** [comment]
please, sync with develop

**Comment by ar4s** [comment]
:ok:

**Pull request merged**